bigip/resource_bigip_ltm_monitor.go

```diff
@@ -185,11 +185,9 @@ func resourceBigipLtmMonitorRead(d *schema.ResourceData, meta interface{}) error
 			if err := d.Set("destination", m.Destination); err != nil {
```

Comment:
in fact err checking is also not needed for `destination`, since it's also a primitive type (`TypeString`)